packages/react-instantsearch-hooks-dom/src/ui/RefinementList.tsx

```diff
@@ -50,17 +52,21 @@ export function RefinementList({
                   }}
                 />
                 <span className="ais-RefinementList-labelText">
-                  <Highlight
-                    parts={[
-                      getHighlightedParts(unescape(item.highlighted || '')),
-                    ]}
-                    classNames={{
-                      root: 'ais-Highlight',
-                      highlighted: 'ais-Highlight-highlighted',
-                      nonHighlighted: 'ais-Highlight-nonHighlighted',
-                      separator: 'ais-Highlight-separator',
-                    }}
-                  />
+                  {searchBox && query.length > 0 ? (
```

Comment:
We just need to check for a query to allow highlighting, right?

```suggestion
                  {query.length > 0 ? (
```

Reply:
Addressed a07c6ede